src/helm/benchmark/run_specs/audio_run_specs.py

```diff
@@ -49,6 +50,12 @@ def get_machine_translation_metric_specs() -> List[MetricSpec]:
     return [MetricSpec(class_name="helm.benchmark.metrics.machine_translation_metrics.MachineTranslationMetric")]
 
 
+def _get_open_ended_generation_metric_specs() -> List[MetricSpec]:
+    return get_basic_metric_specs(
+        ["exact_match", "quasi_exact_match", "f1_score", "rouge_l", "bleu_1", "bleu_4", "cider"]
```

Comment:
Are these metrics typically use for LibriSpeech?

Reply:
Thanks for the reminder! Just added several popular metrics for the audio recognition task